images/macos/toolsets/toolset-12.json

```
@@ -12,19 +12,26 @@
     "xamarin": {
         "vsmac": "latest",
         "mono-versions": [
-            "6.12.0.162"
+            "6.12.0.174"
         ],
         "ios-versions": [
-            "15.8.0.0", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
+            "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
         ],
         "mac-versions": [
-            "8.8.0.0", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
+            "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
         ],
         "android-versions": [
-            "12.0.0.3", "11.3.0.4"
+            "12.2.4.3", "12.0.0.3", "11.3.0.4"
         ],
         "bundle-default": "6_12_14",
```

I think we can update this one as xamarin android is not coupled with the xcode version
```suggestion
        "bundle-default": "6_12_15",
```